R/plot_recurrent_fusions.R

```diff
@@ -12,18 +12,31 @@
 #'
 #' @examples
 #' out_annofuse <- system.file("extdata", "PutativeDriverAnnoFuse_test_v14.tsv", package = "annoFuse")
-#' sfc <- read.delim(out_annofuse)
-#' plot_recurrent_fusions(sfc, groupby = "Fusion_Type", countID = "Sample")
+#' sfc <- read_tsv(out_annofuse)
+#' clinical<-read_tsv(system.file("extdata", "pbta-histologies.tsv", package = "annoFuse"))
+#' # Select only in-frame and frameshift
+#' sfc <- sfc %>%
+#' dplyr::filter(Fusion_Type %in% c("in-frame","frameshift"))
+#' sfc <- merge(sfc ,clinical[,c("Kids_First_Biospecimen_ID","broad_histology")],by.x="Sample",by.y="Kids_First_Biospecimen_ID")
```

Comment:
Also: can be that R CMD check complains about too long lines - because they are not rendered correctly in the help page

What about 

```
sfc <- merge(sfc , clinical[,c("Kids_First_Biospecimen_ID","broad_histology")],
                      by.x="Sample", by.y="Kids_First_Biospecimen_ID")
```

Reply:
ok, the monospaced font is of course rendered differently - trying to edit now

Reply:
Ok, edit's done now 😉